endpoint.go

```diff
@@ -759,10 +768,6 @@ func (ep *endpoint) sbLeave(sb *sandbox, force bool, options ...EndpointOption)
 		return err
 	}
 
-	if e := ep.deleteServiceInfoFromCluster(sb, "sbLeave"); e != nil {
```

Comment:
without this one will it still work the network disconnect?

Reply:
check the corresponding moby/moby PR for this